### ISSUE TITLE
Accuracy of identity matrix assumption

### ISSUE THREAD

**Issue by DanNBullock** [brain-life/validator-neuro-anat#7]
https://github.com/brainlife/validator-neuro-anat/blob/b7c5142e4bc480350da31e637c2a8a362d0fa5b6/run.py#L48-L57

I could certainly be wrong about this, but I don't think it is the case that T1 images _necessarily_ have an identity matrix here.  For example, it could be (for whatever reason) slightly non-orthogonal, and thus have non-zero entries on the off diagonal.  Perhaps this is undesirable, and that's what the warning is trying to nudge about?  Even so, I believe that it is quite possible for T1 images to have sub-millimeter resolutions (i.e. .75), and it would be strange to nudge them (via warning) about a supererogatory characteristic.

**Comment by soichih** [comment]
I agree and I always thought it's odd that we have to make this assumption, but I was told in the past that without affine being ident subsequent processing might fail. I think LAS vs RAS is also the same type of assumption. Ideally, all of our apps should work equally well with data being either LAS or RAS.. 

This is a very old assumption, so if we know that all of our App works well with non ident affine, then we can just drop this warning.
